src/ProjectTemplates/scripts/startvs.cmd

```diff
@@ -0,0 +1,34 @@
+@ECHO OFF
```

Comment:
This is an improvement in testing templates and a bit that I was missing to debug the results of `*Locally` scripts. It could be a separate PR if really needed.

Reply:
Nah, I trust you it works, and if it doesn't, nothing really broke :)